guides/source/api_documentation_guidelines.md

```diff
@@ -162,6 +162,29 @@ On the other hand, regular comments do not use an arrow:
 #   polymorphic_url(record)  # same as comment_url(record)
 ```
 
+### SQL
+
+When documenting SQL statements, the result should not have `=>` before the output.
```

Comment:
```suggestion
When documenting SQL statements, the result should have  `#` before the output.
```

Reply:
@vipulnsward So fast! :zap: I was thinking because the previous section talks about prefixing output with the hash-rocket, that calling it out explicitly here that is **not needed** would be effective.